src/app/(views)/github/t/merge/page.tsx

```diff
@@ -1,6 +1,28 @@
 'use client'
 
+import { Repositories } from '@/components/github/root/repositories/repositories'
+
+import {
+  ALL_INSTALLATION,
+  useFilterChange
+} from '@/contexts/github/root/filter-provider/filter-provider'
+import { useRepos } from '@/contexts/github/root/selected-pulls-provider'
+
 // just using a client component not to think too much about it
 export default function Page() {
-  return <div>Merge page</div>
+  const { repos } = useRepos()
+  const { filterValue } = useFilterChange()
+  const filteredRepos =
+    filterValue === ALL_INSTALLATION
+      ? repos
+      : repos?.filter((repo) => `${repo.installationId}` === filterValue)
```

Comment:
If you want to change ``${repo.installationId}`` to string, then just wrap it with `String` like `String(repo.installationId)`

Reply:
I have a question.
We already merged the `${repo.installatinId}` issue in feat(cff-11). But when we need to make further modifications, what is the best approach to re-merge(modify?) it?

Reply:
I'm not saying it is wrong. I meant that is redundant. The purpose of using "`" is not converting number to string. It's just allowing multi-line string, string interpolation with embedded expression. If we already merged that grammar somewhere, change that one as well because again.

Reply:
I got it. If we need to modify that code anyway, how can I edit the already merged cff-11? Should I create a new issue, or should I restore branch(?) the previous merge for modifications?

Reply:
Just change it in this PR. It's okay